src/lib.rs

```diff
@@ -56,7 +56,7 @@ pub async fn scheduled(event: ScheduledEvent, env: Env, _ctx: ScheduleContext) {
     let mut client = discord::Client::new(token);
     commands::update(&env, &mut client).await;
 
-    if event.cron() != "TEST".to_string() {
+    if event.cron() != "TEST" {
```

Comment:
`cargo clippy --fix` turned this into
```rust
    if event.cron() != *"TEST"
```
on my end. I wonder why it does that specifically. `"TEST"` appears to work perfectly fine.